index.html

```diff
@@ -4,18 +4,31 @@
     <title>Tic Tac Toe!</title>
     <meta charset="utf-8">
     <link href="index.css" media="screen" rel="stylesheet" type="text/css">
+    <link href='https://fonts.googleapis.com/css?family=Indie+Flower' rel='stylesheet' type='text/css'>
   </head>
 
   <body>
     <h1>Tic Tac Toe</h1>
-    <div id="tic-tac-toe"></div>
+    <div id="tic-tac-toe">
+      <div id="row-0-col-0", class="board"></div>
+      <div id="row-0-col-1", class="board"></div>
+      <div id="row-0-col-2", class="board"></div>
+      <br>
+      <div id="row-1-col-0", class="board"></div>
+      <div id="row-1-col-1", class="board"></div>
+      <div id="row-1-col-2", class="board"></div>
+      <br>
+      <div id="row-2-col-0", class="board left-margin"></div>
```

Comment:
Right?! I have noooo idea why. It's pretty slight, so I didn't even notice until a classmate checked out my game...then it bothered her so much that she fixed it for me :P
Thanks for the code review!